roles/loki/tasks/main.yml

```diff
@@ -0,0 +1,119 @@
+---
+# tasks file for loki
+
+- name: Update apt repo and cache on all Ubuntu box
+  apt:
+    update_cache: yes
+    force_apt_get: yes
+    cache_valid_time: 3600
+
+- name: Install required packages
+  apt:
+    name: "{{ install_packages }}"
+    state: present
+  tags:
+    - install_packages
+
+- name: Create loki user
+  user:
+    name: loki
+    create_home: false
+
+- name: Create conf directory
+  file:
+    state: directory
+    path: /etc/loki
+    mode: '0755'
+    group: loki
+    owner: loki
+    recurse: yes
+
+- name: Create data directory
+  file:
+    state: directory
+    path: /data/loki
```

Comment:
I would suggest:
a) to make this configurable (not essential, but for future playbooks)
b) to change this to `/var/lib/loki` as `/data` is not a standard place on UNIX to put data.

Reply:
changed in the latest commit
https://github.com/ipfs-search/ipfs-search-deployment/commit/0df2ecf01c22ce909362e8374859d57a4d5a6fd4

now we can specify the path in a playbook